<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastWidget</class>
 <widget class="QWidget" name="ContrastWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Display Contrast</string>
  </property>
  <widget class="QLabel" name="minValueLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minValueEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="maxValueLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxValueEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="colorBandLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Band:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAllBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRedBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbGreenBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbBlueBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="linearButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="autoStretchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Stretch</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="gammaButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamma</string>
   </property>
  </widget>
  <widget class="QSlider" name="gammaSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="gammaValEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="EdtQtPlotView" name="Plot" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetLinearButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Linear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>411</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EdtQtPlotView</class>
   <extends>QWidget</extends>
   <header>EdtQtPlotView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ContrastWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
